packages/sampler/stories/default/Routable.js

```diff
@@ -0,0 +1,78 @@
+import {action} from '@enact/storybook-utils/addons/actions';
+import React from 'react';
+import Heading from '@enact/ui/Heading';
+import {Linkable, Routable, Route} from '@enact/ui/Routable';
+import {storiesOf} from '@storybook/react';
+
+const LinkedButton = Linkable('button');
+const viewStyle = {padding: '6px', border: '1px solid black'};
+const buttonStyle = {margin: '6px', padding: '6px'};
+
+const AppView = () => (
+	<div style={viewStyle}>
+		<Heading>/app</Heading>
+		<LinkedButton style={buttonStyle} path="./settings">Settings</LinkedButton>
+		<LinkedButton style={buttonStyle} path="./home">Home</LinkedButton>
+	</div>
+);
+
+const SettingsView = () => (
+	<div style={viewStyle}>
+		<Heading>/app/settings</Heading>
+		<LinkedButton style={buttonStyle} path="./wifi">Wi-Fi</LinkedButton>
+		<LinkedButton style={buttonStyle} path="./wired">Wired</LinkedButton>
+		<LinkedButton style={buttonStyle} path="./bluetooth">Bluetooth</LinkedButton>
+	</div>
+);
+const HomeView = () => (<Heading style={viewStyle}>/app/home</Heading>);
+
```

Comment:
Is there any rule to add one empty line between `const` command?
There is one empty line before `SettingsView`, but there is no empty line before `HomeView`.

Reply:
I intended to divide each level. 
Do you prefer to add empty line on each view?

Reply:
It's fine if there is any rule. Let go ahead.